schema/resolvers.js

```diff
@@ -119,7 +125,57 @@ const Resolvers = {
     deleteOption: (_, args, ctx) =>
       ctx.repositories.Option.remove(args.input.id),
     undeleteOption: (_, args, ctx) =>
-      ctx.repositories.Option.undelete(args.input.id)
+      ctx.repositories.Option.undelete(args.input.id),
+    createOtherAnswer: async (root, args, ctx) => {
+      const parentAnswer = await ctx.repositories.Answer.get(
+        args.input.parentAnswerId
+      );
+
+      if (!validMultipleChoiceAnswer(parentAnswer)) {
+        throw new Error(
+          `Answer with id ${args.input
+            .parentAnswerId} must be Radio or Checkbox.`
+        );
+      }
+
+      if (!isNil(parentAnswer.otherAnswerId)) {
+        throw new Error(
+          "Cannot add a second otherAnswer. Delete the existing one first."
+        );
+      }
+
+      const otherAnswer = await ctx.repositories.Answer.createOtherAnswer(
+        parentAnswer
+      );
+
+      return otherAnswer;
```

Comment:
to keep resolvers simple/clean, can we move as much logic as possible into Answer repository? (both for this and delete other answer)